integration-tests/src/test/java/io/quarkiverse/openapi/generator/it/PetStoreTest.java

```diff
@@ -0,0 +1,24 @@
+package io.quarkiverse.openapi.generator.it;
+
+import static io.restassured.RestAssured.when;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.common.QuarkusTestResource;
+import io.quarkus.test.junit.QuarkusTest;
+
+@QuarkusTestResource(WiremockPetStore.class)
+@QuarkusTest
+public class PetStoreTest {
+
+    @Test
+    public void testGetPetById() {
+        final String petName = when()
```

Comment:
Not sure what you're testing here. Shouldn't the REST Client be injected instead of using rest-assured?

Reply:
It was a copy and paste from the later test inside the `deployment` module. I'll change.